manifests/params.pp

```diff
@@ -31,6 +31,11 @@
     $rh_certificate_consumer_host_key = undef
   }
 
+  $ruby_foreman_scap_client_package = $operatingsystem ? {
+    /Debian|Ubuntu/ => ruby-foreman-scap-client,
```

Comment:
Please use quotes in strings.

Reply:
added $osfamily and added quotes.
Thanks a lot for reviewing my changes.